src/index.html

```diff
@@ -12,6 +12,7 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
+    <h1>Moyo header</h1>
```

Comment:
in this task, you need to implement only the header, there should be nothing else